recipes/pylexique/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pylexique" %}
+{% set version = "1.0.7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pylexique-{{ version }}.tar.gz
+  sha256: 53d0150623b149dca768c1f6820c4dc551f78074fa6b7691f1e5e91c702995eb
+
+build:
+  number: 0
+  entry_points:
+    - pylexique=pylexique.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - pytest-runner
+    - python
+  run:
+    - click
+    - cython
```

Comment:
Why do you need `cython` at runtime?

Reply:
Oups, I forgot to remove it.

I will do right now.

Thanks @isuruf